.env

```diff
@@ -74,3 +74,7 @@ ROUTEMASTER_MONITOR_INTERVAL=10
 ROUTEMASTER_SCALING_THRESHOLD=100
 ROUTEMASTER_SCALING_DEADLINE=1000
 
+# maximum number of bytes in an event's optional data payload (when messagepack'ed)
+# DO NOT INCREASE THIS - using the event bus as a data bus is a strong smell of
+# an incompatible architecture, and will cause major bus scalability issues.
+ROUTEMASTER_MAX_EVENT_DATA=64
```

Comment:
I don't think we should be renaming fields -- IMHO this stuff should still obey good semantic principles even if it is a payload, and should still stick to the 'normal' naming conventions.

Regarding no links in the payload... it sounds like what you're saying is that you'd effectively have to merge the 'main' response with the payload. That sounds like it would be confusing to me because you're then effectively saying the client has to know that the 'main' response is static apart from the payload which isn't very RESTful. It also makes the processing of things rather harder. I'm not really a big fan of this idea.